pkg/kubelet/kubelet.go

```diff
@@ -396,6 +396,14 @@ func NewMainKubelet(kubeCfg *componentconfig.KubeletConfiguration, kubeDeps *Kub
 		myConfigureCBR0 = false
 	}
 
+	clusterDNS := make([]net.IP, 0, len(kubeCfg.ClusterDNS))
+	for _, ipstring := range kubeCfg.ClusterDNS {
+		ip := net.ParseIP(ipstring)
+		if ip != nil {
+			clusterDNS = append(clusterDNS, ip)
+		}
```

Comment:
I actually spend a few hours debugging a broken ClusterDNS IP (IP + space instead of IP)....
PR is updated. Good point.